docs/source/quick-start.rst

```diff
@@ -0,0 +1,136 @@
+Quick Start
+===========
+
+Install NuPIC
+-------------
+
+.. code-block:: bash
+
+   pip install nupic [--user]
+
+The ``--user`` is in brackets because it is optional. See the
+`pip reference guide <https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-user>`_
+for details.
+
+The output of this command should end with:
+
+    Successfully installed nupic-X.X.X nupic.bindings-X.X.X
+
+If your installation was unsuccessful, you can find help on
+`NuPIC Forums <https://discourse.numenta.org/c/nupic/>`_ and on
+`Github <https://github.com/numenta/nupic/issues/>`_.
+
+Choose Your API
+---------------
+
+See the `OPF <guide-opf.html>`_ and `Network API <guide-network-api.html>`_
+sections of the `Guide <guide.html>`_.
+
+OPF
```

Comment:
Nowhere else on this page is "OPF" defined.  Might be helpful to expand to "Online Prediction Framework (OPF)".